src/rpc/mining.cpp

```diff
@@ -1064,7 +1064,10 @@ static RPCHelpMan estimatesmartfee()
 {
     RPCTypeCheck(request.params, {UniValue::VNUM, UniValue::VSTR});
     RPCTypeCheckArgument(request.params[0], UniValue::VNUM);
-    unsigned int max_target = ::feeEstimator.HighestTargetTracked(FeeEstimateHorizon::LONG_HALFLIFE);
+
+    CBlockPolicyEstimator& feeEstimator = EnsureFeeEstimator(request.context);
```

Comment:
```suggestion
    CBlockPolicyEstimator& fee_estimator = EnsureFeeEstimator(request.context);
```

snake case for new code ;)

Reply:
Thanks! Amended